include/libopencm3/stm32/h7/memorymap.h

```diff
@@ -154,4 +154,7 @@
 #define TIM3_BASE           0x40000400U
 #define TIM2_BASE           0x40000000U
 
+/* APB-D Peripherals */
+#define DBGMCU_BASE         0x5C001000
```

Comment:
again, not part of this commit, but totally should come in of course.  however, I think the comment is wrong?

`They are accessible to the debugger via the APB-D bus at base address 0xE00E1000. They are also
accessible by the processor core at base address 0x5C001000.` 

 (just drop the comment maybe?)

Reply:
Made comment generic...there are a few other Debug/Trace periphs in the same range that may get added later...I feel weird having no separator at all since it's not an APB1 peripheral :-)